src/test/java/ru/training/at/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,204 @@
+        softAssertObject.assertTrue(displayedName.isDisplayed());
+
+        //SoftAsserting that the displayed username matches the expected username
+        String expectedUsername = "ROMAN IOVLEV";
+        softAssertObject.assertEquals(displayedName.getText(), expectedUsername);
+    }
+
+    public void checkHeaderItems() {
+        //Locating the elements with CSS selectors
+        WebElement homeButton = webDriver.findElement(By.cssSelector("ul.uui-navigation>li:nth-child(1)>a[href='index.html']"));
+        WebElement contactFormButton = webDriver.findElement(By.cssSelector("ul.uui-navigation>li:nth-child(2)>a[href='contacts.html']"));
+        WebElement serviceButton = webDriver.findElement(By.cssSelector("ul.uui-navigation>li:nth-child(3).dropdown>a"));
+        WebElement metalsAndColorsButton = webDriver.findElement(By.cssSelector("ul.uui-navigation>li:nth-child(4)>a[href='metals-colors.html']"));
+
+        // SoftAsserting the visibility of the elements
+        softAssertObject.assertTrue(homeButton.isDisplayed());
+        softAssertObject.assertTrue(contactFormButton.isDisplayed());
+        softAssertObject.assertTrue(serviceButton.isDisplayed());
+        softAssertObject.assertTrue(metalsAndColorsButton.isDisplayed());
+
+        // SoftAsserting button texts
+        softAssertObject.assertEquals(homeButton.getText(), "HOME");
+        softAssertObject.assertEquals(contactFormButton.getText(), "CONTACT FORM");
+        softAssertObject.assertEquals(serviceButton.getText(), "SERVICE");
+        softAssertObject.assertEquals(metalsAndColorsButton.getText(), "METALS & COLORS");
+    }
+
+    public void checkIndexPageIcons() {
+        //Locating elements with CSS selectors
+        WebElement microscopeIcon = webDriver.findElement(By.cssSelector("div.benefit-icon>span.icons-benefit.icon-practise"));
+        WebElement headsetIcon = webDriver.findElement(By.cssSelector("div.benefit-icon>span.icons-benefit.icon-custom"));
+        WebElement screenIcon = webDriver.findElement(By.cssSelector("div.benefit-icon>span.icons-benefit.icon-multi"));
+        WebElement rocketIcon = webDriver.findElement(By.cssSelector("div.benefit-icon>span.icons-benefit.icon-base"));
+
+        //SoftAsserting that the icons are displayed
+        softAssertObject.assertTrue(microscopeIcon.isDisplayed());
+        softAssertObject.assertTrue(headsetIcon.isDisplayed());
+        softAssertObject.assertTrue(screenIcon.isDisplayed());
+        softAssertObject.assertTrue(rocketIcon.isDisplayed());
+    }
+
+    public void checkTextUnderIcons() {
+        //Expected texts
+        String expectedTextMicroscope = "To include good practices\nand ideas from successful\nEPAM project";
+        String expectedTextHeadset = "To be flexible and\ncustomizable";
+        String expectedTextScreen = "To be multiplatform";
+        String expectedTextRocket = "Already have good base\n(about 20 internal and\nsome external projects),\nwish to get more…";
```

Comment:
Could be used `List<String>` for the storing expected values

Reply:
I agree. I will use it.